packages/web-frontend/src/containers/Map/Map.js

```diff
@@ -188,38 +135,21 @@ class MapComponent extends Component {
         <TileLayer tileSetUrl={tileSetUrl} />
         <ZoomControl sidePanelWidth={sidePanelWidth} />
         <DemoLand />
-        {currentOrganisationUnit && organisationUnitIsArea(currentOrganisationUnit) && (
-          <InteractivePolygon
-            area={currentOrganisationUnit}
-            organisationUnitChildren={getChildren(currentOrganisationUnit.organisationUnitCode)}
-            isActive
-            {...basicPropsForInteractivePolygon}
-          />
+        {areMeasuresOnTheSameEntityLevel ? (
```

Comment:
Would be good to add a comment here for why we do this. It's not obvious to me why in the case of the visuals, all being on the same entity level is different.